Sources/Model/TransferMethod/HyperwalletTransferMethod.swift

```diff
@@ -24,115 +24,137 @@ public class HyperwalletTransferMethod: NSObject, Codable {
         case countryOfNationality
+        /// The bank account holder's date of birth.
         case dateOfBirth
+        /// The LicenseId of the bank account holder
         case driversLicenseId
+        /// The employer Id of the  bank account holder
         case employerId
+        /// The bank account holder's first name.
         case firstName
+        /// The bank account holder's gender
         case gender
+        /// The bank account holder's government ID number, such as a Social Security Number.
         case governmentId
+        /// The bank account holder's government IdType
         case governmentIdType
+        /// The bank account holder's  Intermediary Bank AccountId
         case intermediaryBankAccountId
+        /// The bank account holder's  Intermediary Bank AddressLine1
         case intermediaryBankAddressLine1
+        /// The bank account holder's  Intermediary Bank AddressLine2
         case intermediaryBankAddressLine2
+        /// The bank account holder's  Intermediary Bank's City
         case intermediaryBankCity
+        /// The bank account holder's Intermediary Bank's  Country
         case intermediaryBankCountry
+        /// The bank account holder's Intermediary Bank Id
         case intermediaryBankId
+        /// The bank account holder's Intermediary Bank Name
         case intermediaryBankName
+        /// The bank account holder's Intermediary PostalCode:
         case intermediaryBankPostalCode
+        /// The bank account holder's Intermediary Bank's  State and Province
         case intermediaryBankStateProvince
+        /// The bank account holder's last name.
         case lastName
+        /// The bank account holder's middle name.
         case middleName
+        /// The bank account holder's mobile number
         case mobileNumber
+        /// The bank account holder's passport Id
         case passportId
+        /// The bank account holder's phone number.
         case phoneNumber
+        /// The bank account holder's postal code.
         case postalCode
+        /// The bank account holder's profile type.
         case profileType
+        /// The bank account holder's state, province or region.
         case stateProvince
+        /// The wire transfer instructions
         case wireInstructions
 
         /// Bank Card related fields
+        ///
+        /// The card brand.
         case cardBrand
+        /// The 16-digit card number.
         case cardNumber
+        /// The bank card type.
         case cardType
+        /// The card security code which is embossed or printed on the card.
         case cvv
+        /// The expiration date for the card (YYYY-MM).
         case dateOfExpiry
 
-        // PayPal account related fields
+        /// PayPal account related fields
+        ///
+        /// The email address associated with the PayPal account.
         case email
 
-        // prepaid card related fields
+        /// prepaid card related fields
+        ///
```

Comment:
No need to add this in document so just // are fine and remove the extra ///

Reply:
Removed extra /